internal/pkg/docker/dockerengine/dockerengine.go

```diff
@@ -201,6 +210,72 @@ func (c DockerCmdClient) Push(ctx context.Context, uri string, w io.Writer, tags
 	return parts[1], nil
 }
 
+func (in *RunOptions) generateRunArguments() []string {
+	args := []string{"run"}
+
+	//Add container name option.
+	if in.ContainerName != "" {
+		args = append(args, "--name", in.ContainerName)
+	}
+
+	// Add port mappings.
+	for hostPort, containerPort := range in.ContainerPorts {
+		args = append(args, "-p", fmt.Sprintf("%s:%s", hostPort, containerPort))
```

Comment:
can we expand these to the full version? `-p` and `-e`? just makes it clearer what is happening